### ISSUE TITLE
Freehold Mythic is not showing

### ISSUE THREAD

**Issue by Mipsenx** [Meivyn/AdventureGuideLockouts#6]
Freehold Mythic is not showing, not sure if its because i also did one on m+
Was Alliance EU
![2018-09-15](https://user-images.githubusercontent.com/43292592/45585019-b17f6700-b8dd-11e8-9e3e-f00160c29da5.png)
![2018-09-15 1](https://user-images.githubusercontent.com/43292592/45585022-bc39fc00-b8dd-11e8-9f3c-3498f510eff1.png)

**Comment by kurathor** [comment]
Same issue here, Freehold does not show up. In 1.1.9 it did.

**Comment by Meivyn** [comment]
Thanks for your report, currently looking into it. 

I changed how the addon assigns flags to EJ tiles in v1.2.0, in a way that it shouldn't need locales anymore and should work on all clients. I'll see on my side if it's working, so I'll know if that's a client issue or not. If not I'll start debugging. If it's related to the client, I'll need more info from you.

**Comment by Meivyn** [comment]
Seems like it's not a client issue. I'll need some time to figure it out.

**Comment by kurathor** [comment]
I found the issue. When you iterate over your instanceData array to replace the instanceID then the Freehold ID get replaced from 1754 to 1001 and then again via Scarlet Halls from 1001 to 311. Changing

```
for k, v in pairs(instancesData) do
    if instanceID == k then
        instanceID = v
    end
end
```
to
```
for k, v in pairs(instancesData) do
    if instanceID == k then
        instanceID = v
        break
    end
end
```
fixed the issue for me.

**Comment by Meivyn** [comment]
Yes, I noticed it, didn't know how to fix it though, I just woke up. But I found a better way to do it.

Thanks for your contribution.

**Comment by Meivyn** [comment]
Fixed in 44b10c8ad7b3fdd11d259922c87db2586c359ce5.
